sources/platform/api_v2/api_v2_reference.apib

```diff
@@ -3081,6 +3081,9 @@ The POST payload is a JSON object or a JSON array of objects to save into the da
 **IMPORTANT:** The limit of request payload size for the dataset is 5 MB. If the array exceeds the size,
 you'll need to split it into a number of smaller arrays.
 
+If the dataset has fields schema defined, the push request can potentialy fail with `400 Bad Request` if any item does not match the schema.
```

Comment:
It'd be worth mentioning what happens with the valid items. I guess none will be inserted, right?